src/foreman/Command.py

```diff
@@ -11,11 +11,14 @@ def startup(self):
             site = directory.split('/')[-2]
             activation_environment = self.find_virtual_environment_activation_file(directory)
 
-            command = f"cd {directory}"
-            command += f" && source {activation_environment}"
-            command += f" && pip install uwsgi && set -m; nohup uwsgi --socket /tmp/{site}.test.sock --wsgi-file wsgi.py &> /dev/null &"
-            subprocess.run(command, shell=True, close_fds=True,
-                           env={'PYTHONPATH': f'{directory}'})
+            if activation_environment is None:
+                self.warn(f"No virtual environment detected, not starting site {site}")
```

Comment:
I recommend you don't just tell people problems. They know their site didn't start. Try telling them solutions like "No virtual environment detected, not starting site {site}. Please register your venv by running `foreman register /path/to/venv`"

Reply:
Fixed